src/handlebars/releases.handlebars

```diff
@@ -82,12 +82,18 @@
                             <div class='small-dl-text'>\{{installer_extension}}</div>
                           </div>
                         </a>
+                        <a href='\{{link}}' class='latest-download-button latest-download-button-grey a-button'>
```

Comment:
whitespace seems off either here or below.

Reply:
I believe that this is indented correctly?

Reply:
See below on lines 89-91 the closing anchor is indented differently to the opening one below the else

Reply:
that's because it's indented inside the if statement?